src/tigris.ts

```diff
@@ -32,7 +32,12 @@ const AuthorizationHeaderName = "authorization";
 const AuthorizationBearer = "Bearer ";
 
 export interface TigrisClientConfig {
-	serverUrl: string;
+	serverUrl?: string;
+	/**
+	 * Use clientId/clientSecret to authenticate production services.
+	 * Obtains at console.tigrisdata.cloud in Applications section
+	 * or by running `tigris create application` CLI command
```

Comment:
Once we support running user's application code how would this command look like. It's irrelevant question for this code but just something that came to my mind